src/python/WMCore/WorkQueue/WorkQueue.py

```diff
@@ -640,23 +641,61 @@ def queueWork(self, wmspecUrl, request=None, team=None):
         work = self.processInboundWork(inbound, throw=True)
         return len(work)
 
-    def addWork(self, requestName, rucioObj=None):
+    def addWork(self, inboundElem, rucioObj=None):
         """
         Check and add new elements to an existing running request,
         if supported by the start policy.
+
```

Comment:
please add reference where to find inboundElem schema. The code in this method uses different attributes and it is unclear which schema for this data structure is used.